### PR TITLE
Feature/staff bios

### PR DESCRIPTION
**Pull request by agnessa**
- Adds staff group and staff member models
- Adds admin interface to edit them
- Split content_controller into tweets / site_dives / post controllers
- Moved content-related files into Content namespace and directory structure (controllers, models, serializers, specs etc)
- Integrates SimpleMDE to edit staff bios in markdown
- Adds `/content/staff_groups` endpoint

```
{
   "data":[
      {
         "name":"Programmers",
         "position":1,
         "staffMembers":[
            {
               "name":"Agnieszka Figiel",
               "position":1,
               "bio":"Agnieszka is working on\r\n* this,\r\n* that\r\n* and another.",
               "smallImageUrl":"/system/content/staff_members/images/000/000/001/small/profile_square.jpg?1517997942"
            }
         ]
      }
   ]
}
```

simple_mde

Partially implements https://github.com/Vizzuality/trase/issues/149

### CODE REVIEW

**Comment by agnessa** [comment]
I'll add the api endpoint and some specs to complete this if the editor looks good.

**Comment by agnessa** [comment]
Currently this format of response - does it work with the position of the staff group nested and duplicated for each staff member from group?
```
{
   "data":[
      {
         "name":"Agnieszka Figel",
         "position":1, # position of this member
         "bio":"Agnieszka is working on\r\n* this,\r\n* that\r\n* and another.",
         "smallImageUrl":"/system/content/staff_members/images/000/000/001/small/profile_square.jpg?1517997942",
         "staffGroup":{
            "name":"Programmers",
            "position":1 # position of this group
         }
      }
   ]
}
```

**Comment by agnessa** [comment]
@tiagojsag @sorodrigo which would be better? the 1st one is currently in place, I would go for the 2nd one because it avoids repetition (group position) but not sure if that's fine for frontend?

1. 

```
{
   "data":[
      {
         "name":"Agnieszka Figel",
         "position":1, # position of this member
         "bio":"Agnieszka is working on\r\n* this,\r\n* that\r\n* and another.",
         "smallImageUrl":"/system/content/staff_members/images/000/000/001/small/profile_square.jpg?1517997942",
         "staffGroup":{
            "name":"Programmers",
            "position":1 # position of this group
         }
      }
   ]
}
```

2. 

```
{  
   "data":[  
      {  
         "name":"Agnieszka Figiel",
         "position":1,
         "bio":"Agnieszka is working on\r\n* this,\r\n* that\r\n* and another.",
         "staffGroupId":1,
         "smallImageUrl":"/system/content/staff_members/images/000/000/001/small/profile_square.jpg?1517997942"
      }
   ],
   "meta":[  
      {  
         "id":1,
         "name":"Programmers",
         "position":1
      }
   ]
}
```

**Comment by agnessa** [comment]
Ready for review

File: spec/support/schemas/staff_groups.json [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+                  ]
+                },
+                "bio": {
+                  "$id": "/properties/data/items/properties/staffMembers/items/properties/bio",
+                  "type": "string",
+                  "title": "The Bio Schema",
+                  "description": "An explanation about the purpose of this instance.",
+                  "default": "",
+                  "examples": [
+                    "Agnieszka is working on\r\n* this,\r\n* that\r\n* and another."
```
Comment (tiagojsag) [inline spec/support/schemas/staff_groups.json]: busy Agnieszka is busy :P

**Pull request merged**